app/services/flowcommerce_spree/import_experience_items.rb

```diff
@@ -7,7 +7,7 @@ def self.run(zone, client: FlowcommerceSpree.client, organization: ORGANIZATION)
       new(zone, client: client, organization: organization).run
     end
 
-    def run
+    def run # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
Why are we disabling metrics?

Reply:
Rubocop was raising this exception locally, so I thought it would be better to disable these to avoid future warnings. The best solution would be to fix the whole service though, but that would have taken considerable more time.